invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/mirador_preview.html

```diff
@@ -0,0 +1,24 @@
+{#
+    Copyright (C) 2020 CERN.
+    Copyright (C) 2020 Northwestern University.
+
+    Invenio-App-RDM is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{%- extends config.PREVIEWER_ABSTRACT_TEMPLATE %}
+
+{% block panel %}
+<div class="mirador-preview">
+  <p>loading...</p>
+   <div class="ui container">
+     <div class="ui grid column">
+         <div class="column">
+          <div id="demo">
+             <script src="/static/js/m3/dist/main.js"></script>
```

Comment:
I don't know how the Mirador player will use the `id`, but if it will replace the content, it will remove this JS. I will move the JS import down after the last `div`.